blog.html

```diff
@@ -0,0 +1,103 @@
+--- 
+title: Mailmeteor Blog - Learn how to send mass emails with Gmail
```

Comment:
```suggestion
title: Mailmeteor Blog - Learn how to mail merge with Gmail
```

Pour rester sur le focus de "mail merge gmail"